### ISSUE TITLE
auto-pst-pdf compilation ending in error, but pdflatex will continue

### ISSUE THREAD

**Issue by tstone-1** [wspr/will2e#9]
During compilation of a rather large document, I run into (`filename.log`):
```
[179.193]
!pdfTeX error: pdflatex (file ./filename-pics.pdf): PDF inclusion: required pa
ge does not exist <30>
 ==> Fatal error occurred, no output PDF file produced!
```
Well, page 30 in `filename-pics.pdf` is present and looks good to me. I'm using `chemnum` which itself relies on `auto-pst-pdf`. During the auxiliary run of `auto-pst-pdf`,  `filename-autopp.log` is created - and deleted once the auxiliary run has finished. I tailed `filename-autopp.log` and found it to end with:
```
! TeX capacity exceeded, sorry [pool size=2849387].
\tl_if_blank:nF ...\exp_after:wN {\use_none:n #1?}
                                                  \scan_stop: \__prg_F_true:...
l.218 ... von etwa \SI{10}{\angstrom} abgebildet.}
                                                  
If you really absolutely need more capacity,
you can ask a wizard to enlarge me.
```
So, the regular compilation will continue even if the auxiliary compilation has ended with an error. The `filename.log` will then itself end with an error message that is rather useless in error analysis, especially because `filename-autopp.log` has already been deleted at this point.

I referred to [this question][1] and ran `initexmf --edit-config-file=pdflatex`, added
```
buf_size = 10000000
extra_mem_bot = 10000000
extra_mem_top = 10000000
main_memory = 10000000
pool_size = 10000000
```
to `%USERPROFILE%\AppData\Roaming\MiKTeX\miktex\config\pdflatex.ini`. Saved the file and ran `initexmf --dump=pdflatex`. However, the problem persists as described. However, when editing `%USERPROFILE%\AppData\Local\Programs\MiKTeX\miktex\config\texmfapp.ini` and setting `buf_size`, `extra_mem_bot`, `extra_mem_top`, `main_memory`, and `pool_size` to 10000000, respectively, the document will compile as expected. However, this file is forcefully 'repaired' by MiKTeX package manager once the distribution is updated.

I then referred to the manual of `auto-pst-pdf` and found that the auxiliary run invokes `latex` instead of `pdflatex`. Thus, I did the described adjustments to `%USERPROFILE%\AppData\Roaming\MiKTeX\miktex\config\latex.ini` and found a solution to the problem.

TL;DR: Shouldn't the whole compilation process end in error once the `auto-pst-pdf` auxiliary run fails?

  [1]: https://tex.stackexchange.com/questions/438902/how-to-increase-memory-size-for-xelatex-in-miktex
